microsoft-edge/devtools-guide-chromium/css/css-overview-tool.md

```diff
@@ -131,6 +141,31 @@ When a CSS codebase becomes complex, keeping track of all the CSS media queries
    ![Microsoft Edge, with the TODO list demo app and DevTools, showing the Sources tool, with the file that contains a media query opened, at the right line](images/css-overview-tool-sources-tool.png)
 
 
+<!-- ------------------------------ -->
+#### Non-simple selectors
+
+The **Non-simple selectors** section of the CSS overview report lists the CSS non-simple selectors found on the webpage.
+
+In CSS, simple selectors are selectors with only a single component (for example, a single id selector or type selector).  These selectors are not combined with other selector components or combinators.  All basic selectors, attributes, and single pseudo-classes and pseudo-elements are siple selectors.
+
+Non-simple selectors are more complicated and tend to match more elements, potentially resulting in long-running **Recalculate Styles** events in the **Performance** tool.  You can try to change these selectors so that they are more specific and match fewer elements, reducing the amount of time the browser takes to calculate styles when:
```

Comment:
```suggestion
Non-simple selectors are more complicated and tend to match more elements, potentially resulting in long-running **Recalculate Styles** events in the **Performance** tool.  You can try to change these selectors so that they are more specific and match fewer elements, reducing the amount of time the browser takes to calculate styles in the following cases:
```